packages/atclient/include/atclient/request_options.h

```diff
@@ -109,6 +118,18 @@ typedef struct atclient_get_atkeys_request_options {
   uint8_t _initialized_fields[1];
 } atclient_get_atkeys_request_options;
 
+/*
+ * 5. Pkam auhtenticate Request Options
+ */
+
+typedef struct atclient_pkam_authenticate_options {
+  char *root_server_host;
+  int root_server_port;
+  char *atserver_host;
+  int atserver_port;
+  uint8_t _initialized_fields[1];
```

Comment:
ports have a range of 0-65535

So instead of using `int`, we could use `uint16_t`